More2Life/PriceButton.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  PriceButton.swift
+//  More2Life
+//
+//  Created by Brendan Kingsford on 8/29/17.
+//  Copyright © 2017 More2Life. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class PriceButton: UIButton {
```

Comment:
If you want this in a storyboard with @ibdesignable then you will want to put this in the Shared framework.

Reply:
Putting @IBDesignable on a class inside the main project requires the storyboard to build the entire project to render the storyboard. Not good.

Reply:
If you don't want IBDesignable then just ignore this.